packages/@react-spectrum/menu/src/MenuTrigger.tsx

```diff
@@ -103,7 +103,8 @@ export const MenuTrigger = forwardRef(function MenuTrigger(props: SpectrumMenuTr
         scrollRef={menuRef}
         placement={initialPlacement}
         hideArrow
-        shouldFlip={shouldFlip}>
+        shouldFlip={shouldFlip}
+        shouldContainFocus>
```

Comment:
won't this break the "tab out of menu" case?

Reply:
see PR description. I changed this to match RAC.

Reply:
ah, sorry, i forgot the description by the time I got here